lib/rakpios-cli-service-deploy

```diff
@@ -13,7 +13,7 @@ do_select_service(){
 
     # get the list of available services
     services=$(jq '.templates[].title' ${DIR_CNF}/portainer_app_template.json | tr -d '"')
-    services=$(echo -e "$services\nPortainer" | sort)
+    #services=$(echo -e "$services\nPortainer" | sort)
```

Comment:
Delete the line if not needed, but add the sort to the previous line so the services are listed in alphabetical order, which I think makes more sense than the random order in the JSON file